gamemode/modules/afk/sv_afk.lua

```diff
@@ -99,6 +99,13 @@ local function BlockAFKTeamChange(ply, t, force)
 end
 hook.Add("playerCanChangeTeam", "AFKCanChangeTeam", BlockAFKTeamChange)
 
+-- Freeze player's salary
+hook.Add("playerGetSalary", "AFKGetSalary", function(ply, amount)
+    if ply:getDarkRPVar("AFK") then
+        return true, "", 0 -- false, DarkRP.getPhrase("salary_frozen"), 0
```

Comment:
Could you get rid of this commented code?

Reply:
Sure, I was not sure about how you would like it to be implemented, if you wanted the player to be noticed that he's not receiving his salary upon payDay()